spec/spec_routes.rb

```diff
@@ -26,7 +26,7 @@ def draw!
 
         localize do
 
-          get :constraints => proc { false }, "*path" => "FooController#foo"
+          get :constraints => proc { false }, "*path" => "foocontroller#foo"
```

Comment:
This had to be changed because of this error in 4.2: `'FooController' is not a supported controller name. This can lead to potential routing problems. See http://guides.rubyonrails.org/routing.html#specifying-a-controller-to-use`